[2023-03-22T16:26:46.957-0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_active manual__2023-03-22T20:26:41.859406+00:00 [queued]>
[2023-03-22T16:26:46.963-0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_active manual__2023-03-22T20:26:41.859406+00:00 [queued]>
[2023-03-22T16:26:46.963-0400] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T16:26:46.963-0400] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-22T16:26:46.963-0400] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T16:26:46.967-0400] {taskinstance.py:1300} INFO - Executing <Task(HttpSensor): is_api_active> on 2023-03-22 20:26:41.859406+00:00
[2023-03-22T16:26:46.978-0400] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'api_dag', 'is_api_active', 'manual__2023-03-22T20:26:41.859406+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/weatherapi_test.py', '--cfg-path', '/var/folders/71/zhs_yc7n05l7c97rnhn92bb80000gn/T/tmp_55ijtzg']
[2023-03-22T16:26:46.979-0400] {standard_task_runner.py:83} INFO - Job 44: Subtask is_api_active
[2023-03-22T16:26:46.980-0400] {standard_task_runner.py:55} INFO - Started process 75420 to run task
[2023-03-22T16:26:46.998-0400] {task_command.py:388} INFO - Running <TaskInstance: api_dag.is_api_active manual__2023-03-22T20:26:41.859406+00:00 [running]> on host sunandas-laptop.local
[2023-03-22T16:26:47.034-0400] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=is_api_active
AIRFLOW_CTX_EXECUTION_DATE=2023-03-22T20:26:41.859406+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-22T20:26:41.859406+00:00
[2023-03-22T16:26:47.035-0400] {http.py:122} INFO - Poking: weather?q=Monroe,GA,US&units=standard&appid=94ca56afa3eee20f7100985fdfe682aa
[2023-03-22T16:26:47.037-0400] {base.py:73} INFO - Using connection ID 'my_api_connection' for task execution.
